src/main/java/core/basesyntax/service/impl/DataReaderFromFile.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.DataReader;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class DataReaderFromFile implements DataReader {

```

Comment:
I chose the name **DataReaderFromFile** instead of **DataReader**, assuming that in the future there may be another interface that will read data not from the file, but from the database, for example.